### PR TITLE
Two new integration tests for query post endpoint

### PR DESCRIPTION
**Pull request by tisnik**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] New integration tests

## Description

Two new integration tests for query post endpoint

## Related Tickets & Documents

- Related Issue #[OLS-158](https://issues.redhat.com//browse/OLS-158)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Unit tests passed locally.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Just `make test-integration`

### CODE REVIEW

File: tests/integration/test_ols.py [hunk 1]
```diff
@@ -59,3 +61,47 @@ def test_raw_prompt(monkeypatch) -> None:
         "query": "test query",
         "response": "test response",
     }
+
+
+def test_post_question_on_unexpected_payload(monkeypatch) -> None:
```
Comment (bparees) [inline tests/integration/test_ols.py:66]: what's with the unused monkeypath arg?

Comment (tisnik) [reply tests/integration/test_ols.py:66]: leftover, bad me

Comment (tisnik) [reply tests/integration/test_ols.py]: fixed

Comment (tisnik) [reply tests/integration/test_ols.py]: @bparees ^^^

**Pull request merged**